### DISCUSSION TITLE
Welcome to mjg-repo Discussions!

### DISCUSSION THREAD

**Discussion by watterle** [vg-mjg/mjg-repo#149] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sharmayu** [comment]
![1602552177856](https://github.com/vg-mjg/mjg-repo/assets/67625878/8b520309-c3c5-481c-80df-06ee4b248e70)

**Comment by chinponya** [comment]
what did he mean by this
also, the real facebook is here https://github.com/vg-mjg/discussions/discussions

**Comment by udbceuewbberi** [comment]
currently working on an a&c project for which I need the internal text assets of the game.
I tried extracting the PC client's ABM-Fold, but AssetStudio crashes (OutOfMemory). What tool or method you are currently using to decrypt and access the game's assets? Thanks

**Reply by chinponya** [reply]
we have one built from scratch specifically for mjs: [asset meido](https://git.honk.li/czen/asset_meido/releases)
though it targets the web version instead
